### ISSUE TITLE
Allow selecting multiple terms from a vocabulary for faceted search

### ISSUE THREAD

**Issue by pwilkins** [mitodl/lore#622]
Currently the LORE UI supports selecting a single term from a vocabulary for faceted search.  You can select a term from as many facets as are displayed, but no more than one term from each vocabulary.  If you have a repository containing four courses, the faceted search allows you to choose one of the courses or all of the courses, but not two of the courses.

(Actually, I thought we used to be able to do this, but it is no longer supported on lore-ci.) 
@pdpinch

**Comment by pdpinch** [comment]
Did you check production, i.e. lore.odl.mit.edu ?

**Comment by pwilkins** [comment]
Same behavior in production.

**Comment by pdpinch** [comment]
There is still some ambiguity of whether this would function as an "or" or as an "and"

@ferdi have you had any more thoughts on this? Jolyon was advocating for an "or" last week, but I don't think you were convinced.

**Comment by noisecapella** [comment]
I believe "and" has always been the behavior so far with facets. Since we don't have any resources which belong to more than one course you would never see the option to facet on two courses. But if you assign a resource two different terms from the same vocabulary you should be able to facet by both.

**Comment by Ferdi** [comment]
Yes, that's currently by design.  We can consider it as enhancement.
